client/js/utils.js

```diff
@@ -97,6 +98,24 @@ function toggleNotificationMarkers(newState) {
 	viewport.toggleClass("notified", newState);
 }
 
+function togglePasswordField(elem) {
+	$(elem).on("click", function() {
```

Comment:
This needs to pass in a more specific selector or direct jquery element, otherwise you will be binding multiple events to the same buttons if you open connect window 5 times (it'll bind 5 times on the button in settings).

Reply:
@xPaw, is this still relevant?